File: backend/src/main/java/org/codevscovid19/stayhappyathome/entity/Emotion.java
package org.codevscovid19.stayhappyathome.entity;

import java.util.Set;

public enum Emotion {
  HAPPY, RELAXED, ACCOMPLISHED, INFORMED, ENERGIZED, AWWW, INSPIRED, ENTERTAINED;

  public static Set<Emotion> getAll() {
    return Set.of(HAPPY, RELAXED, ACCOMPLISHED, INFORMED, ENERGIZED, AWWW, INSPIRED, ENTERTAINED);
  }
}

File: frontend/happy_at_home/lib/models/Emotion.dart
import 'package:dart_json_mapper/dart_json_mapper.dart';

@jsonSerializable
@Json(enumValues: Emotion.values)
enum Emotion {
  HAPPY,
  RELAXED,
  ACCOMPLISHED,
  INFORMED,
  ENERGIZED,
  AWWW,
  INSPIRED,
  ENTERTAINED
}
